### DISCUSSION TITLE
Welcome to machinelearning_ka_chilla Discussions!

### DISCUSSION THREAD

**Discussion by AammarTufail** [AammarTufail/machinelearning_ka_chilla#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Syed-Riaz-Ali** [comment]
My name is Syed Riaz Ali. I am a Mechanical Engineer and currently doing a job in Oil & Gas sector. I am extremely happy to be a part of this community. I still remember it was 28 December 2021 when I joined "Python ka chilla" and completed the whole course in 40 days. It was a very exciting journey and after that chilla I realized that I had to do something for the growth of this community. So, right after the completion of chilla I started teaching the whole course to my office colleague and now some of them are a part of this Machine learning course.

**Comment by Aafaq10** [comment]
I am **Aafaq ur Rehman**, I did Chemical Engineering from Pakistan followed by Ph.D. from South Korea. Currently, I am working as a postdoctoral fellow at the National Insitute of Korea. I was always fascinated by the way computers communicate with humans however due to the big difference between Computer Science and Chemical Engineering, I never had the time to explore the hidden beauty of programming. Recently, I came across one postdoc offer in Chemical Engineering and the key requirement was expertise in Python. Their idea was to connect the experimental part of chemical engineering with coding and produce something very meaningful and very interesting. Due to unable to code a single line, I was rejected from the offer. So I made up my mind that Now I have to learn coding at any cost. My aim is to learn to code and set new trends in Chemical Engineering using python. 

To all of the perspective fellows at Machine Learning Chilla, try to learn code and apply the same thing in your field. Do not go for the certificates instead understand the meaning of each and every line in this course. Good Luck to all of us. 

Regarads
**Aafaq Ur Rehman Ph.D 
Postdoc fellow at KITECH, Cheonan
South Korea**

**Comment by MALIK2k21** [comment]
I am Muhammad Abdullah, a Mechanical Engineer, currently enrolled in Masters Program in South Korean Institute. I have some experience with Machine Learning, Python and Time Series Forecasting. Currently, I am working on a project related to development of AI based methodologies to make Renewable Energy Systems more reliable.
